cmd/platform/deployment-template/show.go

```diff
@@ -50,4 +53,5 @@ var showCmd = &cobra.Command{
 func init() {
 	Command.AddCommand(showCmd)
 	showCmd.Flags().BoolP(showInstanceConfigurations, "", false, "Shows instance configurations")
+	showCmd.Flags().String(format, "deployment", "If deployment is specified deployment_template is populated in the response, If cluster is specified cluster_template is populated in the response. (Defaults to deployment)")
```

Comment:
```suggestion
	showCmd.Flags().String(format, "deployment", "If a deployment is specified, the deployment_template is populated in the response. If a cluster is specified, the cluster_template is populated in the response. (Defaults to deployment.)")
```

(Same comment as higher up.)